src/openms/source/FORMAT/Base64.cpp

```diff
@@ -1,123 +1,107 @@
 // Copyright (c) 2002-present, OpenMS Inc. -- EKU Tuebingen, ETH Zurich, and FU Berlin
 // SPDX-License-Identifier: BSD-3-Clause
-//
-// --------------------------------------------------------------------------
-// $Maintainer: Timo Sachsenberg $
-// $Authors: Marc Sturm, Chris Bielow, Moritz Aubermann $
-// --------------------------------------------------------------------------
+
+#pragma once
 
 #include <OpenMS/FORMAT/Base64.h>
+#include <OpenMS/SYSTEM/SIMDe.h>
 
 #include <QtCore/QList>
 #include <QtCore/QString>
-
-#include <OpenMS/SYSTEM/SIMDe.h>
+#include <OpenMS/FORMAT/Vec.h>
 
 using namespace std;
 
 namespace OpenMS
 {
-  const simde__m128i mask1_ = simde_mm_set1_epi32(0x3F000000); // 00111111 00000000 00000000 00000000
-  const simde__m128i mask2_ = simde_mm_set1_epi32(0x003F0000); // 00000000 00111111 00000000 00000000
-  const simde__m128i mask3_ = simde_mm_set1_epi32(0x00003F00); // 00000000 00000000 00111111 00000000
-  const simde__m128i mask4_ = simde_mm_set1_epi32(0x0000003F); // 00000000 00000000 00000000 00111111
+  const Vec128 mask1_ = simde_mm_set1_epi32(0x3F000000);
+  const Vec128 mask2_ = simde_mm_set1_epi32(0x003F0000);
+  const Vec128 mask3_ = simde_mm_set1_epi32(0x00003F00);
+  const Vec128 mask4_ = simde_mm_set1_epi32(0x0000003F);
```

Comment:
please keep the comments. They help to understand what is going on.
Also the ones below